tests/framework/CodeInterfaceTests/test_generic_interface_custom_out_file.xml

```diff
@@ -36,7 +36,6 @@
       <fileargs arg="aux" extension=".two" type="input"/>
       <!-- THIS outputFile xml node is used to specify a fixed output file (ONLY CSV) from which RAVEN will collect the results -->
       <outputFile>fixed_output.csv</outputFile>
-      <prepend>python</prepend>
```

Comment:
documentation

Reply:
This is not valid. We put prepend in cargs. I just removed it.

Reply:
No I mean... In the user manual, explanation regarding <prepend> is reported. You should remove it.

Reply:
I think I have already removed it in the user manual. I have checked the code and existing interface, both should be ok.